tensorflow/core/kernels/BUILD

```diff
@@ -6864,37 +6864,31 @@ tf_mkl_kernel_library(
 
 tf_mkl_kernel_library(
     name = "mkl_dequantize_op",
-    srcs = [
-        "mkl_dequantize_op.cc",
-    ],
+    srcs = ["mkl_dequantize_op.cc"],
     hdrs = [
         "meta_support.h",
         "reference_gemm.h",
     ],
-    deps = if_mkl(
-        [
-            ":concat_lib_hdrs",
-            ":conv_ops",
-            ":cwise_op",
-            ":eigen_helpers",
-            ":image_resizer_state",
-            ":ops_util",
-            ":pooling_ops",
-            ":quantization_utils",
-            ":quantized_ops",
-            "//tensorflow/core:array_ops_op_lib",
-            "//tensorflow/core:core_cpu",
-            "//tensorflow/core:framework",
-            "//tensorflow/core:lib",
-            "//tensorflow/core:math_ops_op_lib",
-            "//tensorflow/core:nn_ops_op_lib",
-            "//third_party/eigen3",
-            "@gemmlowp",
-            ":transpose_functor",
-            "@mkl_dnn",
-            "//third_party/mkl:intel_binary_blob",
-        ],
-    ),
+    deps = if_mkl([
```

Comment:
With `mkl_deps`, we actually don't need `if_mkl` anymore. Please remove `if_mkl`. Thank you!